mssql/base.py

```diff
@@ -616,6 +664,8 @@ def format_params(self, params):
 
     def execute(self, sql, params=None):
         self.last_sql = sql
+        if 'GROUP BY' in sql:
```

Comment:
should we do the same for executemany ?

Reply:
It'll be difficult to support this logic for multiple queries because we group params if they are the same (and different queries have different params).

In the future we could tighten the constraints for grouping params so that it only applies to case/when expressions.